src/schema.ts

```diff
@@ -198,6 +201,19 @@ export interface operations {
       404: components["responses"]["NotFound"];
     };
   };
+  "get-users-me": {
+    responses: {
+      /** OK */
+      200: {
+        content: {
+          "application/json": {
+            id?: number;
```

Comment:
Why id and name are not required? I think when receiving 200 you have always the data... vice versa... the status is not 200

Reply:
giust